chains/ethereum/listener.go

```diff
@@ -65,18 +69,75 @@ func (l *Listener) GetSubscriptions() []*Subscription {
 // Start registers all subscriptions provided by the config
 func (l *Listener) Start() error {
 	l.log.Debug("Starting listener...")
+	currBlock, err := l.conn.conn.BlockByNumber(l.conn.ctx, nil)
+	if err != nil {
+		return fmt.Errorf("Unable to get starting Block: %s", err)
+	}
+	if currBlock.Number().Cmp(l.cfg.startBlock) < 0 {
+		return fmt.Errorf("starting block (%d) is greater than latest known block (%d)", l.cfg.startBlock, currBlock.Number())
+	}
+
```

Comment:
the listener should start from `cfg.opts.startBlock`, not the latest block on chain

Reply:
`cfg` has no field `opts`, just `startblock`